server/app.es6

```diff
@@ -34,6 +34,8 @@ app.options('*', function(req, res) {
   res.send();
 });
 
+app.use('/api', express.static('api'));
```

Comment:
Can we use a path more descriptive, such as `docs` with the swagger documentation in a swagger directory inside of that?